torch/csrc/jit/tensorexpr/expr.cpp

```diff
@@ -134,6 +146,18 @@ Expr trunc(const Expr& v) {
   return Intrinsics::make(kTrunc, v);
 }
 
+Expr sigmoid(const Expr& v) {
```

Comment:
`sigmoid` isn't an intrinsic, it's implemented as `1.f / (1.f + expf(-${0}))", "1. / (1. + exp(-${0}))`